src/Transform/ONNX/CMakeLists.txt

```diff
@@ -103,3 +103,15 @@ add_onnx_mlir_library(OMONNXDimAnalysis
   MLIRPass
   MLIRTransforms
   )
+
+add_onnx_mlir_library(OMONNXReplaceNoValue
+  ReplaceNoValueOperands.cpp
+
+  LINK_LIBS PUBLIC
+  OMONNXOps
+  OMONNXRewrite
+  OMShapeInference
```

Comment:
Why do you depend on the shape inference lib?

Reply:
This was from an earlier version, I deleted this